packages/protocol/contracts-0.8/common/GasPriceMinimum.sol

```diff
@@ -162,10 +162,8 @@ contract GasPriceMinimum is
       ISortedOracles sortedOracles = ISortedOracles(
         registry.getAddressForOrDie(SORTED_ORACLES_REGISTRY_ID)
       );
-      uint256 rateNumerator;
-      uint256 rateDenominator;
-      (rateNumerator, rateDenominator) = sortedOracles.medianRate(tokenAddress);
-      return ((gasPriceMinimum() * rateNumerator) / rateDenominator);
+      (uint256 medianRate, uint256 denominator) = sortedOracles.medianRate(tokenAddress);
+      return ((gasPriceMinimum() * medianRate) / denominator);
```

Comment:
Could be a good opportunity to replace this line for a remove https://github.com/celo-org/celo-monorepo/blob/release/core-contracts/11/packages/protocol/contracts/stability/SortedOracles.sol#L185